tasks.py

```diff
@@ -48,7 +48,7 @@ def is_truthy(arg):
 namespace.configure(
     {
         "nautobot_data_validation_engine": {
-            "nautobot_ver": "2.0.0",
+            "nautobot_ver": "2.2.0",
```

Comment:
Unless the minimum version of the app is updated to 2.2 this should revert. Will cause problems on data migrations if make migrations is run against 2.2 but installed on <2.2

Reply:
Just to confirm, I thought the matrix figured this part out?

Reply:
For CI yes but the issue is doing local dev against anything other than the minimum version can cause you to fight the ether with CI

Reply:
I wasn't aware of this as a known problem - do we have a writeup about this anywhere? Or can you provide one that we can reference in the future as a best practice / cautionary tale? :-)